examples/tests/external_logging_stream.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+import matplotlib.pyplot as plt
+
+from avocado import Test
+
+
+class MatplotlibTest(Test):
+    def test(self):
+
+        logging.getLogger("matplotlib").level = logging.DEBUG
+        _, ax = plt.subplots()
+
+        fruits = ["apple", "blueberry", "cherry", "orange"]
+        counts = [40, 100, 30, 55]
+        bar_labels = ["red", "blue", "_red", "orange"]
+        bar_colors = ["tab:red", "tab:blue", "tab:red", "tab:orange"]
+
+        ax.bar(fruits, counts, label=bar_labels, color=bar_colors)
+
+        ax.set_ylabel("fruit supply")
+        ax.set_title("Fruit supply by kind and color")
+        ax.legend(title="Fruit color")
```

Comment:
Do we have a place to automatically CI test out examples? Like verify the output here?

Reply:
Here is an alternative minimal example using aexpect:
```python
import logging

from avocado import Test

from aexpect import remote


host="localhost"
username="root"
password="test1234"
client="ssh"
port=22
prompt="^\\[.*\\][\\#\\$]\\s*$"
linesep="\n"
status_test_command="echo $?"


class MatplotlibTest(Test):
    def test(self):

        logging.getLogger("aexpect").level = logging.DEBUG
        logging.getLogger("aexpect").parent = logging.getLogger("avocado.test")
        session = remote.remote_login(client,
                                      host, port, username, password,
                                      prompt, verbose=True)
        session.close()
```
where attaching the logger makes the library output available in the debug log in the correct format but seems to produce some spurious outputs also on stdout.